framework/Metrics/metrics/Factory.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on Jul 18 2016
+
+@author: mandd
```

Comment:
update docstring

Reply:
I didn't change it substantially enough to change authorship; also, as we identified, this authorship note is redundant to git logs and probably should be removed, as many of us have directly modified nearly every file in RAVEN.